planning/behavior_velocity_planner/src/scene_module/occlusion_spot/geometry.cpp

```diff
@@ -59,7 +59,10 @@ void buildSlices(
    * +--------------------------+ - +---+(max_length,min_distance) = inner_polygons
    */
   const double min_length = offset;  // + p.baselink_to_front;
-  const double max_length = std::min(p.detection_area_max_length, p.detection_area_length);
+  // Note: min_detection_area_length is for occlusion spot visualization but not effective for planning
```

Comment:
this is currently for visualization before reaching 
![image](https://user-images.githubusercontent.com/65527974/154180331-9c044a12-d63b-42b7-aacf-e69dfbd7e0ff.png)